src/main/default/lwc/abandonModal/abandonModal.html

```diff
@@ -0,0 +1,48 @@
+<template>
+  <section
+    role="dialog"
+    tabindex="-1"
+    class={modalClass}
+    aria-labelledby="modal-heading"
+    aria-modal="true"
+    aria-describedby="modal-content"
+  >
+    <div class="slds-modal__container">
+      <header class="slds-modal__header">
+        <lightning-button-icon
+          onclick={closeModal}
+          class="slds-modal__close"
+          icon-name="utility:close"
+          variant="bare-inverse"
+          alternative-text={labels.close}
+          title={labels.close}
+        ></lightning-button-icon>
+        <h2 id="modal-heading" class="slds-modal__title slds-hyphenate">
+          {headerLabel}
+        </h2>
+      </header>
+      <div class="slds-modal__content slds-p-around_medium" id="modal-content">
+        <slot name="content">
+          <p class="slds-text-heading_small slds-var-p-around_large">
+            {labels.yourRequestWillNotBeSaved}<br />
+            {labels.thankYouForFeedback}
```

Comment:
Checked with UX and this message is because we are considering abandoning the request a feedback saying that our documentation helped, what do u think? make sense or we should use a less confusing message? x)

Reply:
Hmmm this is just my own opinion but given this is inferred feedback, it might be confusing to a user that hasn't given any explicit feedback. The case itself might have been the feedback. But they canceled it... so they might wonder what really happened.
I think "thanks for your feedback" is great if the user uses the 'like/vote' button a document but here I might leave it out. Totally up for debate though.